src/sentry/integrations/gitlab/repository.py

```diff
@@ -14,20 +14,18 @@ class GitlabRepositoryProvider(providers.IntegrationRepositoryProvider):
 
     def get_installation(self, integration_id, organization_id):
         if integration_id is None:
-            raise ValueError('%s requires an integration_id' % self.name)
+            raise IntegrationError('%s requires an integration id.' % self.name)
 
-        try:
-            integration_model = Integration.objects.get(
-                id=integration_id,
-                organizations=organization_id,
-            )
-        except Integration.DoesNotExist as error:
-            self.handle_api_error(error)
+        integration_model = Integration.objects.get(
+            id=integration_id,
+            organizations=organization_id,
+            provider='gitlab',
+        )
```

Comment:
Will the `DoesNotExist` errors show up in sentry as issues?

Reply:
Currently Yes. should we change it to just logging?

Reply:
I think so. Users can request all sorts of silly things that we don't need to be alerted on.